models/op/token_distributions/optimism/op_token_distributions_optimism_foundation_wallet_approvals.sql

```diff
@@ -51,10 +53,10 @@ FROM {{ source('erc20_optimism', 'evt_Approval') }} a
     LEFT JOIN project_labels al
         ON a.spender = al.address
 WHERE
-    a.contract_address = '{{op_token_address}}' --OP Token
+    a.contract_address = {{op_token_address}} --OP Token
```

Comment:
after seeing the variable usage here, i think the above comment would work to remove single quotes and remove cast to varbinary

Reply:
testing this